packages/core/root/core-root-browser-internal/src/core_system.ts

```diff
@@ -332,6 +332,7 @@ export class CoreSystem {
         injectedMetadata,
         notifications,
         customBranding,
+        core: this.coreContext,
```

Comment:
For consistency, `coreContext` should be passed via the `chrome` service's constructor

e.g.

https://github.com/elastic/kibana/blob/c50f3d749ed9071bf4c0974ed4f71399627f088a/packages/core/root/core-root-browser-internal/src/core_system.ts#L140-L151

Reply:
Good point 👍 Addressed in https://github.com/elastic/kibana/pull/174544/commits/5e09dfe549a6fa39917c8b82a44fa38cd59ae540